fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionAktivitetsbegransningar.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.luae_na.v1.model.converter.certificate.question;
+
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.AKTIVITETSBEGRANSNING_CATEGORY_ID;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.AKTIVITETSBEGRANSNING_DELSVAR_ID_17;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.AKTIVITETSBEGRANSNING_DELSVAR_TEXT;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.AKTIVITETSBEGRANSNING_SVAR_JSON_ID_17;
+import static se.inera.intyg.common.support.facade.util.ValidationExpressionToolkit.singleExpression;
+
+import se.inera.intyg.common.services.texts.CertificateTextProvider;
+import se.inera.intyg.common.support.facade.model.Certificate;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigTextArea;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataTextValue;
+import se.inera.intyg.common.support.facade.util.ValueToolkit;
+
+public class QuestionAktivitetsbegransningar {
+
+    private static final short TEXT_LIMIT = 4000;
+
+    public static CertificateDataElement toCertificate(String textValue, int index, CertificateTextProvider textProvider) {
+        return CertificateDataElement.builder()
+            .id(AKTIVITETSBEGRANSNING_DELSVAR_ID_17)
+            .parent(AKTIVITETSBEGRANSNING_CATEGORY_ID)
```

Comment:
Shouldn't this have the "Header"-question as parent?

Reply:
My last two commits highlights my own confusion regardning this :)

When I used the header as parent, the distance between the header & the question increased. To get closer to how it looks in the angular client, I therefore changed the parent back to category. Im not sure if this should be handled in frontend and we keep the same structure as we did for Dodsplatskommun in Dodsbevis.